sandbox/settings.py

```diff
@@ -133,6 +133,7 @@ class Base(DRFMixin, Configuration):
                     "django.template.context_processors.static",
                     "cms.context_processors.cms_settings",
                     "richie.apps.core.context_processors.site_metas",
+                    "richie.apps.core.context_processors.cdn_domain",
```

Comment:
Actually, you could add it to the `site_metas` context processor... it is exactly the idea behind it.
Adding this context processor to your project is required for Richie to work properly so let's keep it simple and group everything under the same processor?

Reply:
:ok_hand: